README update (#204)
radix-ai/poetry-cookiecutter: README.md

1. Install the latest [Cruft](https://github.com/cruft/cruft) and [Cookiecutter](https://github.com/cookiecutter/cookiecutter) in your [Python environment](https://github.com/pyenv/pyenv-virtualenv) with:
   ```sh
   pip install --upgrade "cruft>=2.12.0" "cookiecutter>=2.1.1"
   ```

> 2. Create a new (empty) repository and clone it locally.

Reviewer: Does the empty matter here? As a reader, it leaves me wondering why that would be important.

Author: The first time I used the template, I thought step 2 referred to the cookie-cutter repo itself. (Also because it wasn't clear directly that the template is applied in step 3, but this part is resolved now.)
I would suggest to make this explicit, but adding `empty` is maybe not clear enough.
Another suggestion: `Create your new repository, then clone it locally`, where 'your' makes it clear it's not about the current cookie-cutter repo.

Reviewer: To my mind, the 'your' refers to something that isn't defined yet. To solve that, how about:
> Create a new repository for your Python project, then clone it locally.

> 3. To apply the Poetry Cookiecutter template, run in the parent directory that contains the cloned repository:

Reviewer: I ran this through ChatGPT to check the copy and it suggested:
```suggestion
3. Run the following command in the parent directory of the cloned repository to apply the Poetry Cookiecutter template:
```

1. Install the latest [Cruft](https://github.com/cruft/cruft) and [Cookiecutter](https://github.com/cookiecutter/cookiecutter) in your [Python environment](https://github.com/pyenv/pyenv-virtualenv) with:
   ```sh
   pip install --upgrade "cruft>=2.12.0" "cookiecutter>=2.1.1"
   ```
2. Create a new repository for your Python project, then clone it locally.
3. Run the following command in the parent directory of the cloned repository to apply the Poetry Cookiecutter template:
   ```sh
   cruft create -f https://github.com/radix-ai/poetry-cookiecutter
   ```

> 4. _Optional:_ if your repository name differs from your project's slugified package name *(see first template parameter below)*, you will need to copy the scaffolded project into the repository with:

Reviewer: We can add a direct link to help them find the parameter docs:
```suggestion
4. _Optional:_ if your repository name differs from your project's slugified package name (see `package_name` in the [Template parameters](https://github.com/radix-ai/poetry-cookiecutter#-template-parameters) below), you will need to copy the scaffolded project into the repository with:
```

| Parameter                                                               | Description                                                                                                                                                                                                                                                                                                                                                                                       |
| ----------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | `package_name` <br> "Spline Reticulator"                                | The name of the package. Will be slugified to `snake_case` for importing and `kebab-case` for installing. Example package name `My Package` will result in `my_package` as sluggified package name (`my-package` for installing).                                                                                                                                                                                                                                      |

Reviewer: Suggestion to make this a bit more compact:
```suggestion
| `package_name` <br> "Spline Reticulator"                                | The name of the package. Will be slugified to `snake_case` for importing and `kebab-case` for installing. For example, `My Package` will be `my_package` for importing and `my-package` for installing.                                                                                                                                                                                                                                      |
```
